src/Tests/AzureClientEntryPointTests.cs

```diff
@@ -81,6 +82,94 @@ public async Task FromSnippet()
             Assert.IsNotNull(job);
         }
 
+        [TestMethod]
+        public async Task QIRSubmission()
+        {
+            var entryPointGenerator = Init("Workspace", new string[] { SNIPPETS.HelloQ });
+            var entryPoint = entryPointGenerator.Generate("HelloQ", null, forceQir: true);
+
+            Assert.IsNotNull(entryPoint);
+            var job = await entryPoint.SubmitAsync(
+                new MockQIRSubmitter(new List<Argument>()),
+                new AzureSubmissionContext());
+            Assert.IsNotNull(job);
+        }
+
+        [TestMethod]
+        public async Task ValidParameterTypes()
+        {
+            var entryPointGenerator = Init("Workspace", new string[] { SNIPPETS.ValidParameterTypes });
+            var entryPoint = entryPointGenerator.Generate("UseValidParameterTypes", null, forceQir: true);
+
+            Assert.IsNotNull(entryPoint);
+
+            var validArguments = new Dictionary<Argument, string>()
```

Comment:
Can you please add test cases in which an entry-point operation receives multiple parameters (mixed-type parameters, same-type parameters)?

Reply:
Not sure what you mean. The test ValidParameterTypes tests a mix of all the valid parameter types, so I believe this is what you are suggesting, right?

Reply:
I see, I did not read the code correctly. Then, in addition to this mixed-types case, can you please add one individual test case per argument type? That way if there's an issue with a specific type it would be easy to know the problematic one.

Reply:
done